h2o-py/h2o/backend/server.py

```diff
@@ -109,6 +110,7 @@ def start(jar_path=None, nthreads=-1, enable_assertions=True, max_mem_size=None,
         assert_is_type(extra_classpath, None, [str])
         assert_is_type(jvm_custom_args, list, None)
         assert_is_type(bind_to_localhost, bool)
+        assert_is_type(extra_args, {"hdfs_config": str})
```

Comment:
I would allow also h2o parameters. So the expected type should be `{str: str}`, IMHO